Formula/pokerstove.rb

```diff
@@ -14,6 +14,7 @@ class Pokerstove < Formula
   end
 
   depends_on "cmake" => :build
+  depends_on "googletest" => :build
```

Comment:
Did you mean to include this?

Reply:
Arg, sorry.  Was working on that separately and it leaked into that commit.  One second...